src/app/(nextjs_migration)/(authenticated)/user/layout.tsx

```diff
@@ -65,7 +65,11 @@ const MainLayout = async (props: Props) => {
         src="/nextjs_migration/client/js/nav.js"
         nonce={getNonce()}
       />
-      <PageLoadEvent userId={userId} />
+      <PageLoadEvent
+        userId={userId}
+        channel={process.env.APP_ENV ?? ""}
```

Comment:
The channel is equivalent to `APP_ENV` for now, I'm not sure we'll ever need or want to change it, but I made it a separate prop just in case.